dashboard/app/models/script_level.rb

```diff
@@ -67,6 +67,15 @@ def end_of_stage?
     stage.script_levels.to_a.last == self
   end
 
+  def long_assessment?
+    if assessment
+      if level["pages"] && level["pages"].length > 1
```

Comment:
er wait, pages is in properties, so I don't know if this works? Do you actually want level.properties["pages"] ?